docs/model_library/strategic_water_management/index.rst

```diff
@@ -1517,69 +1548,69 @@ The water quality at the completions pad intermediate node is dependent on the f
 
 .. math::
 
-    \sum_{n \in N | (n, p) \in NCA}\textcolor{purple}{F_{n,p,t}^{Piped}} \cdot \textcolor{red}{Q_{n,w,t}}
-        + \sum_{\tilde{p} \in P | (\tilde{p}, p) \in PCA}\textcolor{purple}{F_{\tilde{p},p,t}^{Piped}} \cdot \textcolor{green}{\nu_{\tilde{p},w,[t]}}
-        + \sum_{s \in S | (s, p) \in SCA}\textcolor{purple}{F_{s,p,t}^{Piped}} \cdot \textcolor{red}{Q_{s,w,t}}
+    \sum_{n \in N | (n, p) \in NCA}\textcolor{purple}{F_{n,p,t}^{Piped}} \cdot \textcolor{red}{Q_{n,qc,t}}
+        + \sum_{\tilde{p} \in P | (\tilde{p}, p) \in PCA}\textcolor{purple}{F_{\tilde{p},p,t}^{Piped}} \cdot \textcolor{green}{\nu_{\tilde{p},qc,[t]}}
+        + \sum_{s \in S | (s, p) \in SCA}\textcolor{purple}{F_{s,p,t}^{Piped}} \cdot \textcolor{red}{Q_{s,qc,t}}
 
-        + \sum_{\tilde{p} \in P | (\tilde{p}, p) \in CCA}\textcolor{purple}{F_{\tilde{p},p,t}^{Piped}} \cdot \textcolor{green}{\nu_{\tilde{p},w,[t]}}
-        + \sum_{r \in R | (r, p) \in RCA}\textcolor{purple}{F_{r,p,t}^{Piped}} \cdot \textcolor{red}{Q_{r,w,t}}
-        + \sum_{f \in F | (f, p) \in FCA}\textcolor{purple}{F_{f,p,t}^{Sourced}} \cdot \textcolor{red}{Q_{f,w,t}}
+        + \sum_{\tilde{p} \in P | (\tilde{p}, p) \in CCA}\textcolor{purple}{F_{\tilde{p},p,t}^{Piped}} \cdot \textcolor{green}{\nu_{\tilde{p},qc,[t]}}
+        + \sum_{r \in R | (r, p) \in RCA}\textcolor{purple}{F_{r,p,t}^{Piped}} \cdot \textcolor{red}{Q_{r^{TreatedWaterNodes},qc,t}}
```

Comment:
Why does this line have `Q_{r^{TreatedWaterNodes},qc,t}` instead of just `Q_{r,qc,t}`?

Reply:
At most nodes, there is a single water quality associated with the output water at that node. However, at treatment sites there is the quality associated with the mixed inlet (R),  treated water ( R-PostTreatmentTreatedWaterNode) and quality associated with the residual ( R-PostTreatmentResidualWaterNode). They are technically from the same node R, but to differentiate the qualities we append the tags. We have previously used the same logic for completions pad + completions pad storage sites. 
See lines here for their definitions: https://github.com/project-pareto/project-pareto/blob/84050a74dcdbd638b1b655056ad937fc94bf6b78/pareto/strategic_water_management/strategic_produced_water_optimization.py#L5265